Tasks/GoTool/task.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "334727F4-9495-4F9D-A391-FC621D671474",
+    "name": "GoTool",
+    "friendlyName": "Go Tool Installer",
+    "description": "Finds or Downloads and caches specified version spec of Go and adds it to the PATH.",
+    "helpMarkDown": "",
+    "category": "Tool",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
```

Comment:
preview: true right?

Reply:
Critical - release this in preview.  You can't go from released to preview

Reply:
Done.

Reply:
Will take care of it

Reply:
Done
changes incorporated in https://github.com/Microsoft/vsts-tasks/pull/6307